src/cpp/src/continuous_batching_pipeline.cpp

```diff
@@ -95,29 +114,31 @@ class ContinuousBatchingPipeline::Impl {
 
         DeviceConfig device_config(core, scheduler_config, device, plugin_config);
 
-        apply_paged_attention_transformations(model, device_config);
+        apply_paged_attention_transformations(model, device_config, true);
```

Comment:
why do we need per layer by default? 
I supposed on current stage we will have the same block tables for each layer, won't  we? 
Anyway, I think per_layer should be set only for use_cache_eviction, right?

Reply:
Right, I'll put it under a condition